.gitattributes

```
@@ -14,15 +14,16 @@
 tool/ci.sh linguist-generated=true
 
 ## Generated by Pigeon
-**/pigeons/*.java linguist-generated=true
+*Pigeon.java linguist-generated=true
+*Messages.java linguist-generated=true
```

I think this needs to be:

```suggestion
**/*Pigeon.java linguist-generated=true
**/*Messages.java linguist-generated=true
```

since `*` matches everything except a slash, this would only catch top-level files.